current/en-us/4. binding/6. delegate-vs-trigger.md

```diff
@@ -19,6 +19,8 @@ To find out if [event delegation](https://davidwalsh.name/event-delegate) can be
 
 Here's the [MDN page for blur](https://developer.mozilla.org/en-US/docs/Web/Events/blur). It has further info on event delegation techniques for the blur and focus events.
 
+When using `delegate` you might also want to use the [`self` binding behavior](/docs/binding/binding-behaviors#self).
```

Comment:
I think this may accidentally give wrong impression that either
- delegate should be accompanied by self
- only delegate can work with self

Maybe we can word it in a way that signals `self` is complementary to event, instead of enhancement. Something like this:

```
Aurelia provides a few binding behaviors to help define advance event handling behavior out of the box, please see
1...
2...
3...
```

@EisenbergEffect @fkleuver

Reply:
Ah, yes of course, `self` can be used also for `trigger`! Sorry, I'm a little confused...

What other binding behaviors would be in that list?